provider/docker/config_service_test.go

```diff
@@ -9,6 +9,63 @@ import (
 	docker "github.com/docker/docker/api/types"
 )
 
+func TestDockerGetFuncMapLabel(t *testing.T) {
```

Comment:
Could you please improve your test to add cases with mutliple custom request/response headers ?

Reply:
@mmatur Done